examples/presigned-url-example.ts

```diff
@@ -58,13 +58,14 @@ async function getSigningKey(): Promise<string> {
     'Did not find signing key in environment, creating new signing key'
   );
   const signingKeyResponse = await momento.createSigningKey(60 * 24); // valid for 24 hours
+  let returnVal = '';
```

Comment:
i'm sorry to keep nitpicking about this.  You can avoid this mutable var and the unfortunate code path related to it if you just add a final `else` clause to the condition and then throw from it, I think that's better for example code.